db/migrate/20240105140341_migrate_geojson_string_to_json.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class MigrateGeojsonStringToJson < ActiveRecord::Migration[7.0]
```

Comment:
Shall we rename this migration to be more specific ?

Reply:
I've followed the pattern with others, although happy to make it `MigrateGeojsonStringFromPlanningApplicationConstraintsQueryToJson`